src/media-segment-request.js

```diff
@@ -298,18 +298,6 @@ const decryptSegment = (decrypter, segment, doneFn) => {
   ]);
 };
 
-/**
- * The purpose of this function is to get the most pertinent error from the
- * array of errors.
- * For instance if a timeout and two aborts occur, then the aborts were
- * likely triggered by the timeout so return that error object.
- */
-const getMostImportantError = (errors) => {
```

Comment:
- Does this change mean that we might not get the most "pertinent" error on the player? 
- Or does it actually end up returning the most relevant error by bubbling up the first error?
- Is there a case where we would want other requests in the group to be made even if one failed?

Reply:
Technically, before this change I don't think it was possible for there to be more than just aborted/the original error. So this should have no change.